.circleci/config.yml

```diff
@@ -310,7 +300,10 @@ workflows:
           filters:
             branches:
               only:
+                - dev
+                - stage
                 - test
+                - qa/**
```

Comment:
added this in so that it will run on any branch with `qa/<branch-name>` so that we can run the tests locally and not worry about pushing up to test and having to cherry pick any thing.  i think this will be good to help with updates and other things that might be done